cloudinstancecredentials-config

```diff
@@ -0,0 +1,40 @@
+#! /usr/bin/python3
+
+# Copyright 2024 SUSE LLC
+#
+# This file is part of cloudinstancecredentials
+#
+# cloudinstancecredentials is free software: you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# cloudinstancecredentials is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# cloudinstancecredentials. If not, see <http://www.gnu.org/licenses/>.
+
+import logging
+from cloudinstancecredentials import (
+    frameworkfactory,
+    config
+)
+
+# PROCESS
+log = logging.getLogger('config-credentials')
+metadata = frameworkfactory.get_metadata(log)
+
+if not metadata:
+    log.warning('No instance metadata. Process finished.')
+    quit()
```

Comment:
commented outside this PR, if this is intended for prod, we should use `sys.exit([value])` as `sys` module is guaranteed to be present while `quit()` depends on `site` module